internal/k8s/controllers/service_controller.go

```diff
@@ -47,6 +47,9 @@ type ServiceReconciler struct {
 	Endpoints         NeedEndPoints
 	LoadBalancerClass string
 	Reload            chan event.GenericEvent
+	// initialLoadPerformed turns true after the first reprocessAll. while false don't reconcile services
```

Comment:
```suggestion
	// initialLoadPerformed is set after the first time we call reprocessAll. This is required because we want the first time we load the services to follow the assigned first, non assigned later order.
	// This allows avoiding to have services with already assigned IP to get their IP stolen by other services.
```

Reply:
done